build.gradle.kts

```diff
@@ -54,53 +52,53 @@ kotlin {
         publishLibraryVariantsGroupedByFlavor = true // This line
     }
 
-    if (System.getenv("SDK_NAME")?.startsWith("iphoneos") == true) {
-        iosArm64("ios") {
-            println("iosArm64")
-            compilations.getByName("main") {
-                val indylib by cinterops.creating {
-                    defFile(project.file("../ssi-mobile-sdk/indylib/indylib.def"))
-                    extraOpts("-libraryPath", "$projectDir/indylib")
-                    extraOpts("-compiler-options", "-std=c99 -I$projectDir/indylib")
-                }
-            }
-            binaries.all {
-                linkerOpts("-L$projectDir/socketlib", "-lPocketSocket")
-            }
-        }
-    } else {
-        iosX64("ios") {
-            println("iosX64")
-            compilations.getByName("main") {
-                val indylib by cinterops.creating {
-                    defFile(project.file("../ssi-mobile-sdk/indylib/indylib.def"))
-                    extraOpts("-libraryPath", "$projectDir/indylib")
-                    extraOpts("-compiler-options", "-std=c99 -I$projectDir/indylib")
-                }
-            }
-            binaries.all {
-                linkerOpts("-L$projectDir/socketlib", "-lPocketSocket")
-            }
-        }
-    }
-
-    cocoapods {
-        pod("PocketSocket") {
-            source = git("https://github.com/zwopple/PocketSocket") {
-                tag = "1.0.1"
-            }
-        }
-
-        summary = "Kotlin sample project with CocoaPods dependencies"
-        homepage = "https://github.com/Kotlin/kotlin-with-cocoapods-sample"
-        ios.deploymentTarget = "12.2"
-        osx.deploymentTarget = "10.8"
-        tvos.deploymentTarget = "9.0"
-
-        frameworkName = "ssi_agent"
-        podfile = project.file("./samples/swiftIosApp/Podfile")
-
-    }
+//    if (System.getenv("SDK_NAME")?.startsWith("iphoneos") == true) {
```

Comment:
I disabled the ios part, because of the upgrade to the newest macOS. It turned out that it is XCode (a really strange error described on the Internet) is not compatible, I will turn it on after delivering the IOS WS part.